parquet-pig/src/main/java/org/apache/parquet/pig/ParquetLoader.java

```diff
@@ -169,6 +185,7 @@ private void setInput(String location, Job job) throws IOException {
     getConfiguration(job).set(PARQUET_PIG_SCHEMA, pigSchemaToString(schema));
     getConfiguration(job).set(PARQUET_PIG_REQUIRED_FIELDS, serializeRequiredFieldList(requiredFieldList));
     getConfiguration(job).set(PARQUET_COLUMN_INDEX_ACCESS, Boolean.toString(columnIndexAccess));
+    SerializationUtil.writeObjectToConfAsBase64(ParquetInputFormat.FILTER_PREDICATE, getFromUDFContext(ParquetInputFormat.FILTER_PREDICATE), getConfiguration(job));
```

Comment:
I stole this line from somewhere else, but this looks like a better approach.

I'll update.